DBMS2/Assignment 1 and 2.sql

```diff
@@ -0,0 +1,60 @@
+-- ################################# Assignment 1 ################################# --
```

Comment:
Where is this 'write a SQL query for that returns a Resultset containing Zip Code, City Names and States ordered by State Name and City Name.
'????